src/components/app/exporters/YnabAccountMappingTable.vue

```diff
@@ -47,28 +47,48 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
+import Vue, { PropType } from 'vue';
 import { required } from '@/components/shared/formValidations';
+import { ref } from '@vue/composition-api';
+import { cloneDeep } from 'lodash';
+
+const objToArrayOfKeyValueArray = (obj: Record<string, string>) => Object.keys(obj).map((key) => ({ key, value: obj[key] }));
```

Comment:
This code seems way more generic and unclear than it should be.

This is a very specific use case with specific values. The generic names and types here just make it harder to understand what is going on.